.github/workflows/presubmit-review.yml

```
@@ -0,0 +1,28 @@
+  pull-requests: write
+  issues: write
+
+on:
+  pull_request:
+  pull_request_review_comment:
+    types: [opened, synchronize]
+
+jobs:
+  Review:
+    runs-on: ubuntu-latest
+      - name: Check required secrets
```

actionlint (1.7.3)

could not parse as YAML: yaml: line 16: mapping values are not allowed in this context

(syntax-check)